pilot/pkg/xds/fake.go

```diff
@@ -51,6 +51,9 @@ import (
 	"istio.io/istio/pkg/test"
 )
 
+// DiscoveryClient is a client interface for XDS.
+type DiscoveryClient = discovery.AggregatedDiscoveryService_StreamAggregatedResourcesClient
```

Comment:
can you keep this in ads.go? its more generic than fake

Reply:
sure